stake/order.py

```diff
@@ -40,6 +40,16 @@ class Config:
         alias_generator = camelcase
 
 
+class Brokerage(BaseModel):
```

Comment:
Could the above brokerage class from the order module be reused in this case?

Reply:
actually scratch that, i have now remembered that the ASX module is mostly a copy of this